src/NUnitFramework/framework/Compatibility/ReflectionExtensions.cs

```diff
@@ -99,415 +97,127 @@ public static Delegate CreateDelegate(this MethodInfo method, Type type)
     public static class TypeExtensions
     {
         /// <summary>
-        /// Returns an array of generic arguments for the give type
+        /// Returns an array of <see cref="Type"/> objects that represent the type arguments of a closed generic type or the type parameters of a generic type definition.
         /// </summary>
-        /// <param name="type"></param>
-        /// <returns></returns>
         public static Type[] GetGenericArguments(this Type type)
         {
-            var info = type.GetTypeInfo();
-            return info.GenericTypeArguments.Concat(info.GenericTypeParameters).ToArray();
+            return type.GetTypeInfo().GetGenericArguments();
```

Comment:
We have an extension for Type.GetTypeInfo() for .NET 2.0 and other frameworks that don't support it that just returns Type for those frameworks. Because of that, we added GetTypeInfo to nearly every instance of type. So, now that PCL is gone, do we even need these extensions?

Reply:
@rprouse We do not, but they are a convenience if you don't like typing `GetTypeInfo()`. I don't have a strong opinion either way on whether we keep these shortcuts.

Reply:
I don't have a strong opinion either, it just seems like these extension methods are now very thin wrappers and that the standard now is to call `.GetTypeInfo()` on types, so might be more expected. Let's merge as is and get a feel for the shape of the code.

Reply:
👍 They did a 180 with `GetTypeInfo()` in .NET Standard 2.0, so I'm wondering if it won't end up being considered legacy.